internal/events/event_stream_producer.go

```diff
@@ -27,9 +27,7 @@ type EventStreamSender struct {
 func (esp *EventStreamSender) RaiseEvent(eventType string, payload []byte, headers []kafka.Header) error {
 	logging.Log.Debugf("publishing message %v to topic %q...", eventType, EventStreamTopic)
 
-	producerConfig := kafka.ProducerConfig{Topic: config.KafkaTopic(EventStreamTopic)}
-	kafkaConfig := kafka.Config{KafkaBrokers: config.KafkaBrokers, ProducerConfig: producerConfig}
-	kf := &kafka.Manager{Config: kafkaConfig}
+	kf := kafka.GetKafkaProducer(config.KafkaSaslSettings, config.KafkaBrokers, EventStreamTopic)
```

Comment:
Technically for the first 2 args you can just do `config.Get()` in the kafka package - what would you think about just having this use that and only accept the one argument (the topic we need to get a producer for) and for consumer the topic + groupid? 

I know it probably ties it more to our `config/` package - but we're already using the kafkabrokers/sasl settings struct so we may as well use `config.Get()` as well.

Reply:
I thought of that, but I was concerned about the reusability. The [superkey-worker uses `source-api-go`'s `kafka/` module](https://github.com/RedHatInsights/sources-superkey-worker/blob/master/main.go#L47-L53). So in that case, if we use `GetKafkaProducer`, it would try to load the `sources-api-go`'s config, which may or may not fail, depending on the env vars that are passed to the superkey worker and the resulting JSON file.

Reply:
Oh yeah great point. We can definitely just leave it the way it is then! My only thing is maybe we could have the brokers be the first argument? That's just me thinking through it too much though.